tools/3Dtrees_standardization/.shed.yml

```
@@ -0,0 +1,9 @@
+name: 3Dtrees_standardization
```

```suggestion
name: 3dtrees_standardization
```

lets keep all lower letters

---

yes